lessons/basics/modules.md

```diff
@@ -194,9 +202,28 @@ import List, only: :functions
 import List, only: :macros
 ```
 
+It is also worth noting that import is lexical in scope.
```

Comment:
Its straight out of the documentation:

```
iex(1)> h import

                         defmacro import(module, opts)

Imports functions and macros from other modules.

import/2 allows one to easily access functions or macros from others modules
without using the qualified name.
```

That said, if you dont like it I can see why.